plugins/woocommerce/src/Internal/Admin/Settings/Payments.php

```diff
@@ -175,6 +175,39 @@ function ( $a, $b ) {
 		return $payment_providers;
 	}
 
+	/**
+	 * Get the business location country code for the Payments settings.
+	 *
+	 * @return string The ISO 3166-1 alpha-2 country code to use for the overall business location.
+	 *                If the user didn't set a location, the WC base location country code is used.
+	 */
+	public function get_country(): string {
+		$user_nox_meta = get_user_meta( get_current_user_id(), self::USER_PAYMENTS_NOX_PROFILE_KEY, true );
+		if ( ! empty( $user_nox_meta['location'] ) ) {
```

Comment:
In line with the renaming in JS, could we use `business_country_code` instead of `location`? It would make it more transparent what it is about and what format.